src/mrinufft/operators/interfaces/cufinufft.py

```diff
@@ -523,7 +523,7 @@ def __adj_op(self, coeffs_d, image_d):
             ret = self.raw_op.type1(coeffs_d, image_d)
         return ret
 
-    def data_consistency(self, image_data, obs_data):
+    def get_grad(self, image_data, obs_data):
```

Comment:
DId we fixate finally on this name?

Reply:
we decided on `data_consistency` indeed.

Reply:
This is addressed in the other PR